### PR TITLE
some ambient and hostile entities

### PR DESCRIPTION
**Pull request by Kahzerx**

### CODE REVIEW

File: mappings/18w15a#18w16a.tinydiff [hunk 1]
```diff
@@ -3052,11 +3052,11 @@ c	net/minecraft/unmapped/C_5587144$C_6955547
 		p	4		p_4	
 		p	5		p_5	
 	m	(Lnet/minecraft/unmapped/C_2137190;Lnet/minecraft/unmapped/C_3674802;Lnet/minecraft/unmapped/C_2441996;Lnet/minecraft/unmapped/C_2441996;Lnet/minecraft/unmapped/C_2441996;)Z	m_4699966		canPlaceBlock
-		p	1			p_1
-		p	2			p_2
-		p	3			p_3
-		p	4			p_4
-		p	5			p_5
+		p	1			worldView
```
Comment (SpaceWalkerRS) [inline mappings/18w15a#18w16a.tinydiff:3055]: `World` implements various interfaces, and variables of those types can still just be called `world`

File: mappings/18w15a#18w16a.tinydiff [hunk 2]
```diff
@@ -4355,9 +4355,9 @@ c	net/minecraft/unmapped/C_7618048
 	m	(Lnet/minecraft/unmapped/C_5553933;Lnet/minecraft/unmapped/C_3674802;)Z	m_4842222	m_4842222	
 		p	1		p_1	
 		p	2		p_2	
-	m	(Lnet/minecraft/unmapped/C_9844932;Lnet/minecraft/unmapped/C_3674802;)V	m_1063774		m_1063774
-		p	1			p_1
-		p	2			p_2
+	m	(Lnet/minecraft/unmapped/C_9844932;Lnet/minecraft/unmapped/C_3674802;)V	m_1063774		tickStepping
+		p	1			worldAccess
```
Comment (SpaceWalkerRS) [inline mappings/18w15a#18w16a.tinydiff:4359]: same as above

**Review by SpaceWalkerRS — CHANGES_REQUESTED** [review]

**Comment by Kahzerx** [comment]
changed

**Review by SpaceWalkerRS — APPROVED** [review]
excellent work

**Comment by SpaceWalkerRS** [comment]
I'll merge it in once the workflow is done

**Comment by SpaceWalkerRS** [comment]
uh-oh, looks like the `canSpawn` method leads to a conflict

**Comment by SpaceWalkerRS** [comment]
the conflict exists at least in 1.8, so you might want to look at it there

**Review by SpaceWalkerRS — APPROVED** [review]

**Pull request merged**